### PR TITLE
subbing out a file for githubteacher-reommendations

### LINKED ISSUE #18: TDYsebade - Visit Sawtelle
Visit Sawtelle (Little Osaka) :+1: (25 minute drive with light traffic)
- [ ] 405 north, exit National, turn left, immediately turn right onto Sawtelle
- [ ] Follow Sawtelle for a few miles, park at the Micheals/PetsMart center on the right (after Pico) for free
- [ ] Eat ramen at Daikokuya/Tsujita/Tatsu
- [ ] Milk tea boba at Coco/Volcano Tea House
- [ ] Buy snacks from Nijiya Market
- [ ] Karaoke

### PR DESCRIPTION
**Pull request by TDYsebade**
This is a much longer message! :neckbeard: :hankey: 

This resolves #18

### CODE REVIEW

File: githubteacher-recommendations.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Fuquay-Varina, NC
+
+### Places to Eat
```
Comment (githubteacher) [inline githubteacher-recommendations.md:3]: I like Italian - do you have any recommendations?

Comment (githubteacher) [inline githubteacher-recommendations.md:3]: @TDYsebade what is the local food everyone needs to try? Fish Tacos?

Comment (TDYsebade) [inline githubteacher-recommendations.md:3]: Check issue #18 for my reccomendation

File: githubteacher-recommendations.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+### Fun things to do
+
+### Places to Stay
```
Comment (kmazh) [inline githubteacher-recommendations.md:11]: @letitcv Great

File: githubteacher-recommendations.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+## Fuquay-Varina, NC
+
+### Places to Eat
+
+### Fun things to do
```
Comment (kmazh) [inline githubteacher-recommendations.md:8]: Fun is in the air

**Pull request merged**